src/plugins/controls/public/options_list/components/options_list_popover_action_bar.tsx

```diff
@@ -52,23 +53,20 @@ export const OptionsListPopoverActionBar = ({
     <div className="optionsList__actions">
       <EuiFormRow fullWidth>
         <EuiFlexGroup className="optionsList__searchSortRow" gutterSize="xs" responsive={false}>
-          <EuiFlexItem grow={true}>
-            <EuiFieldSearch
-              isInvalid={!searchString.valid}
-              compressed
-              disabled={showOnlySelected}
-              fullWidth
-              onChange={(event) => updateSearchString(event.target.value)}
-              value={searchString.value}
-              data-test-subj="optionsList-control-search-input"
-              placeholder={OptionsListStrings.popover.searchPlaceholder[
-                searchTechnique ?? OPTIONS_LIST_DEFAULT_SEARCH_TECHNIQUE
-              ].getPlaceholderText()}
-            />
-          </EuiFlexItem>
-          {!hideSort && (
-            <EuiFlexItem className="optionsList__sortButtonWrapper" grow={false}>
-              <OptionsListPopoverSortingButton showOnlySelected={showOnlySelected} />
+          {fieldSpec?.type !== 'date' && (
```

Comment:
**very very small nit:** It looks like hiding the search bar for date fields adds a **little** bit too much space at the top of the popover to me 😅 

![image](https://github.com/elastic/kibana/assets/8698078/549f9324-2128-4148-8f76-e4e52dd0b06f)

I think if we move the `fieldSpec?.type !== 'date'` check up to surround the entire `EuiFlexGroup` this would be resolved.

Reply:
Oh yeah we don't actually need the `EuiFlexGroup` anymore since I moved the sort button. I just had to tweak some CSS to add top padding to the search bar, which ends up looking like this.

<img width="483" alt="Screenshot 2023-09-26 at 3 06 49 PM" src="https://github.com/elastic/kibana/assets/1697105/3e2deb33-4fcd-4d42-89e1-dd7eb04c457b">

<img width="749" alt="Screenshot 2023-09-26 at 3 07 44 PM" src="https://github.com/elastic/kibana/assets/1697105/eb097407-04f3-46d4-b967-ea56bbadef72">